<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>80</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MPU-9250</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="IMU6500GB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>770</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>680</width>
      <height>271</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>770</width>
      <height>640</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>IMU 6500</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Accelerometer    ±</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Gyroscope   ±</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>66</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>106</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PlotAccelerometerPB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Plot Signals</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PlotGyroscopePB">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Plot Signals</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>290</y>
       <width>331</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>331</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(°/second)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="GyroscopeCB">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>250</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="AccelerometerCB">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>55</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(g's)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>106</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>66</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_AccelerometerX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>62</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_AccelerometerY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_AccelerometerZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_GyroscopeX">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>62</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_GyroscopeZ">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_GyroscopeY">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>100</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AKGB">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>370</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>271</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>370</width>
      <height>640</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>AK 8963</string>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Magnometer (µT)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PlotMagnometerPB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Plot Signals</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>331</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>106</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>66</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_MagnetometerY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_MagnetometerX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>62</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_MagnetometerZ">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TempGB">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Temperature Sensor</string>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>86</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Temperature:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TempValue">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TempModePB">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Change to Farenheit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TemperatureMode">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ArduinoSettingsGB">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Arduino Settings</string>
    </property>
    <widget class="QComboBox" name="DelayCB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Delay (ms):</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_arduinostatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Arduino not connected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ConnectArduinoPB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Connect to Arduino</string>
     </property>
    </widget>
    <widget class="QComboBox" name="BaudCB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Data Rate (baud):</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuREAD_ME">
    <property name="title">
     <string>READ ME</string>
    </property>
    <addaction name="actionMPU_9250_Data_Sheet"/>
   </widget>
   <addaction name="menuREAD_ME"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMPU_9250_Data_Sheet">
   <property name="text">
    <string>MPU-9250 Data Sheet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
